bucket/pdftk-builder.json

```diff
@@ -0,0 +1,25 @@
+{
+    "homepage": "http://www.angusj.com/pdftkb/",
+    "description": "A free graphical interface to PDFTK. Take control of your PDF documents - merge or split pages; stamp pages or add background watermarks; add page numbers; and even rotate pages.",
+    "version": "3.10.0",
+    "license": "GPL-3.0-only",
+    "url": "http://www.angusj.com/pdftkb/pdftkb_setup.exe",
+    "hash": "c6eacc1e93185d2e8608dbae643267c893108b699dfef22bcda885b484fafc86",
+    "innosetup": true,
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\PDFTKBuilder.ini\")) { New-Item \"$dir\\PDFTKBuilder.ini\" | Out-Null }"
+    ],
+    "shortcuts": [
+        [
+            "PDFTKBuilder.exe",
+            "PDFTK Builder"
```

Comment:
Shortcut with this name will be created by installer?

Reply:
Yes, the name is splited by a space.

Reply:
https://github.com/lukesampson/scoop-extras/pull/3056#discussion_r338548603